src/olympia/users/models.py

```diff
@@ -1303,6 +1307,19 @@ class BannedUserContent(ModelBase):
     addons = models.ManyToManyField('addons.Addon')
     addons_users = models.ManyToManyField('addons.AddonUser')
     ratings = models.ManyToManyField('ratings.Rating')
+    picture_backup_name = models.CharField(
+        max_length=75, default=None, null=True, blank=True
+    )
+    picture_type = models.CharField(max_length=75, default=None, null=True, blank=True)
+
+    def restore_avatar(self):
```

Comment:
I prefer avatar to picture too but we should be consistent - i.e. either rename it to avatar everywhere or call this `restore_picture` (or `restore_user_picture`)

Reply:
3a95da5eba17fd57ad6db2943743ac076d5d0731